.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "deps/onig"]
+	path = deps/onig
```

Comment:
That file was autogenerated by git and I don't see the two-space convention used anywhere else inside `.gitmodules` in the atom project. If you think it's still worth doing, then sure, no problem.